lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -0,0 +1,246 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import namedtuple
+from ryu.ofproto.ofproto_v1_4 import OFPP_LOCAL
+
+from .base import MagmaController
+from magma.pipelined.openflow import flows
+from magma.pipelined.openflow.magma_match import MagmaMatch
+from magma.pipelined.app.inout import INGRESS
+from ryu.lib.packet import ether_types
+from magma.pipelined.app.base import MagmaController, ControllerType
+
+GTP_PORT_MAC = "02:00:00:00:00:01"
+
+class Classifier(MagmaController):
+    """
+    A controller that sets up an openflow pipeline for Magma.
+    This controller is used for install the default tunnel entry,
+    add tunnel flows for gtp_port/local_port/mtr_port and delete
+    tunnel flows for gtp_port/local_port/mtr_port into OVS table 0.
+    """
+    APP_NAME = "classifier"
+    APP_TYPE = ControllerType.SPECIAL
+    ClassifierConfig = namedtuple(
+            'ClassifierConfig',
+            ['gtp_port', 'uplink_port_name', 'mtr_ip', 'mtr_port'],
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(Classifier, self).__init__(*args, **kwargs)
+        self.config = self._get_config(kwargs['config'])
+        self.tbl_num = self._service_manager.get_table_num(self.APP_NAME)
+        self.next_table = self._service_manager.get_table_num(INGRESS)
+        self._uplink_port = OFPP_LOCAL
+        self._cookie = 1
+        self._datapath = None
+
+    def _get_config(self, config_dict):
+        port_name = None
+        mtr_ip = None
+        mtr_port = None
+        
+        if 'ovs_uplink_port_name' in config_dict:
+            port_name = config_dict['ovs_uplink_port_name']
+
+        if 'mtr_ip' in config_dict:
```

Comment:
Can you also add the port/default flow from this commit https://github.com/magma/magma/commit/8db5af3d38d0d7368ec7de0c996067a5dc72c18c?